features/resource-timing.yml

```diff
@@ -0,0 +1,27 @@
+name: Resource timing (initial support)
+description: The `serverTiming` property of the `PerformanceResourceTiming` API contains server timing information about network requests.
```

Comment:
There's no `serverTiming` in the BCD keys below, but lots of other things on `PerformanceResourceTiming`. Is this description still correct?

Reply:
Oops, I copied from https://github.com/web-platform-dx/web-features/blob/main/features/server-timing.dist.yml to match its style, then wrote the description into the dist, and clobbered it. How about…

```suggestion
description: `PerformanceResourceTiming` entries report when network events happen while loading a resource, such as when connections start and end. You can use this information to measure loading times.
```

Reply:
I've never used this feature, but that looks reasonably accurate!